nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/QueryDatabaseTableRecord.java

```diff
@@ -44,7 +44,7 @@
 import java.util.List;
 import java.util.Set;
 
-import static org.apache.nifi.processors.standard.util.JdbcProperties.USE_AVRO_LOGICAL_TYPES;
+import static org.apache.nifi.util.db.JdbcProperties.USE_AVRO_LOGICAL_TYPES;
```

Comment:
I need to add the registry-only properties to this processor and change QueryDatabaseTable to use the registry-only properties as well. This will involve moving the registry-only properties out of the metrics utils class and into JdbcProperties